internal/bcdb/worldstate_query_processor.go

```diff
@@ -140,3 +143,58 @@ func (q *worldstateQueryProcessor) getNodeConfig(nodeID string) (*types.GetNodeC
 
 	return c, nil
 }
+
+func (q *worldstateQueryProcessor) executeJSONQuery(dbName, querierUserID string, query []byte) (*types.DataQueryResponse, error) {
```

Comment:
> We'll have to add support for that later, and it would be good not to change the API when that happens, just expand it.

which API? the SDK API exported to the user can support limit and skip now also. When we support `limit` and `skip`, we need to change the SDK. I prefer to do it incrementally. 

design have kept all these in mind. Code is easily extendable.

Reply:
Done.

Reply:
Yes, I meant that the structure will be at least:
```
{
  "selector" : {...}
}
```
The rest could follow later.